File: Programing EF DbContext/AfterCompletingChapter7/Model/Address.cs
using System.ComponentModel.DataAnnotations;

namespace Model
{
  [ComplexType]
  public class Address
  {
    public int AddressId { get; set; }
    [MaxLength(150)]
    [Column("StreetAddress")]
    public string StreetAddress { get; set; }
    [Column("City")]
    public string City { get; set; }
    [Column("State")]
    public string State { get; set; }
    [Column("ZipCode")]
    public string ZipCode { get; set; }
  }
}

File: Programming EF Code First/chapter5/Chapter 5 VB/Data Annotations/Model/Address.vb
Imports System.ComponentModel.DataAnnotations

Namespace Model
  <ComplexType>
  Public Class Address
	Public Property AddressId() As Integer
	<MaxLength(150), Column("StreetAddress")>
	Public Property StreetAddress() As String
	<Column("City")>
	Public Property City() As String
	<Column("State")>
	Public Property State() As String
	<Column("ZipCode")>
	Public Property ZipCode() As String
  End Class
End Namespace